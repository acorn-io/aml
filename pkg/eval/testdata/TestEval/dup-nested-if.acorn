a: {
	b: {
		x: 1
	}
	if b.x == 1 {
		y: 2
	}
	if y == 2 {
		z: 1
	}
	if z == 1 {
		q: 1
	}
}
b: {
		y: 1
		if a.y == 1 {
			x: 2
		}
    b: a.b.x
}