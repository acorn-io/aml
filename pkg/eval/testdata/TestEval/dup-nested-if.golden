{
  "a": {
    "b": {
      "x": 1
    },
    "q": 1,
    "y": 2,
    "z": 1
  },
  "b": {
    "b": 1,
    "y": 1
  }
}